daras_ai_v2/language_model_settings_widgets.py

```diff
@@ -32,16 +32,24 @@ def language_model_selector(
     )
 
 
-def language_model_settings(selected_model: str = None):
-    try:
-        llm = LargeLanguageModels[selected_model]
-    except KeyError:
-        llm = None
+def language_model_settings(selected_models: str | list[str] | None = None) -> None:
+    if isinstance(selected_models, str):
+        selected_models = [selected_models]
+    elif not selected_models:
+        selected_models = []
+
+    llms = []
+    for model in selected_models:
+        try:
+            llms.append(LargeLanguageModels[model])
+        except KeyError:
+            pass
 
     col1, col2 = gui.columns(2)
     with col1:
         gui.checkbox("Avoid Repetition", key="avoid_repetition")
-    if not llm or llm.supports_json:
+
+    if any(map(lambda llm: llm.supports_json, llms)):
```

Comment:
we support json output artificially via prompting for models that don't support it natively, so we might wanna leave this like we had earlier?